docs/canvas/canvas-share-workpad.asciidoc

```diff
@@ -0,0 +1,34 @@
+[role="xpack"]
+[[workpad-share-options]]
+== Share your workpad
+
+When you are ready to share your workpad, create a PDF, or export your workpad.
+
+[float]
+[[create-workpad-pdf]]
+=== Create a PDF
+
+To view your workpad outside of Kibana, generate a PDF. 
+
+. If you are using a basic license, enable reporting in your `config/kibana.yml` file.
```

Comment:
Reporting isn't available on the Basic license, only Gold or Platinum. You won't be able to enable it in the `kibana.yml`, so we should put a disclaimer here that this isn't available when you have a basic.

Reply:
Maybe this section should go under the export a workpad section since it's not available for all users?

Reply:
Since these are two separate processes, I'd like to keep them separate, but I'll add a note.

Reply:
Do users actually need to enable reporting before they create a PDF?

Reply:
Reporting is available only in Gold and Platinum, and I think reporting is enabled by default, but if reporting is disabled, then we'll show a dialog in the app to enable reporting in their `kibana.yml`.